autodiscern/annotations.py

```diff
@@ -0,0 +1,237 @@
+from allennlp.data.tokenizers.word_tokenizer import WordTokenizer
+from allennlp.predictors.predictor import Predictor
+from bs4 import BeautifulSoup
+from bs4.element import Tag
+import re
+from typing import Callable, Dict, List, Tuple
+from autodiscern.data_manager import DataManager
+
+
+def add_word_token_annotations(inputs: Dict[str, Dict]) -> Dict[str, Dict]:
+    tok = WordTokenizer()
+    for id in inputs:
+        # have to convert tokens to text because spacy tokens are not pickleable
+        inputs[id]['tokens'] = [t.text for t in tok.tokenize(inputs[id]['content'])]
+    return inputs
+
+
+def add_metamap_annotations(inputs: Dict[str, Dict], dm: DataManager, metamap_path: str = None) -> Dict[str, Dict]:
+    from pymetamap import MetaMapLite
+
+    if metamap_path is None:
+        print("NOTE: no Metamap path provided. Using Laura's default")
+        metamap_path = '/Users/laurakinkead/Documents/metamap/public_mm_lite/'
```

Comment:
For people on Windows OS, the forked [MetamapLite](https://github.com/orisenbazuru/pymetamap) would require to specify `git_bash_pth`. By default it assumes it is installed in `'C:\Program Files\Git\git-bash.exe'`. It is not that issue; one option is to add `git_bash_pth: str = None` as keyword argument. I am fine either ways.

Reply:
fixed